tests/test_vasp_util.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+import os
+import sys
+import numpy as np
+from pymatgen.io.vasp.inputs import Poscar
+from pymatgen.io.vasp.outputs import Vasprun
+from flare.struc import Structure, get_unique_species
+from flare.dft_interface.vasp_util import parse_dft_forces, run_dft, \
+    edit_dft_input_positions, dft_input_to_structure, md_trajectory_from_vasprun
+
+def cleanup_vasp_run(target: str = None):
+    os.system('rm POSCAR')
+    os.system('rm vasprun.xml')
+
+# ------------------------------------------------------
+#                   test  otf helper functions
+# ------------------------------------------------------
+
+@pytest.mark.parametrize("poscar",
+                         [
+                             ('./test_files/test_POSCAR')
+                         ]
+                         )
+def test_structure_parsing(poscar):
+    structure = dft_input_to_structure(poscar)
+    pmg_struct = Poscar.from_file(poscar).structure
+    assert len(structure.species_labels) == len(pmg_struct)
+    assert (structure.cell == pmg_struct.lattice.matrix).all()
+    for i, spec in enumerate(structure.species_labels):
+        assert spec == pmg_struct[i].specie.symbol
+
```

Comment:
Could you also add a line to assert the positions were parsed right?

Reply:
Yup, added this in `test_structure_parsing`.